addon/classes/Column.js

```diff
@@ -129,11 +130,37 @@ export default class Column extends EmberObject.extend({
 
   /**
    * The minimum width (in px) that this column can be resized to.
-   * @property minResizeWidth
+   * @property _minResizeWidth
    * @type {Number}
    * @default 0
+   * @private
+   */
+  _minResizeWidth: 0,
+
+  /**
+   * The minimum width that this column can be resized to.
+   * @property minResizeWidth
+   * @type {String}
+   * @default '0px'
    */
-  minResizeWidth: 0,
+  minResizeWidth: computed({
+    get() {
+      return this.get('_minResizeWidth');
+    },
+    set(key, value) {
+      if (typeOf(value) === 'string') {
+        let [, quantity, units] = value.match(/(\d+)(px)?/) || [];
+        if (units !== 'px') {
+          warn('`value` attribute is interpreted as px regardless of provided units', {
+            id: 'ember-light-table.classes.Column'
+          });
+        }
+        this.set('_minResizeWidth', Number(quantity));
```

Comment:
```js
return this.set('_minResizeWidth', Number(quantity));
```

> I wrote a unit test for the work, but I am unsatisfied with it because I had to do `col.get('_minResizeWidth')` and not `col.get('minResizeWidth')` because the latter was returning undefined. Any idea why that could be?

This is why. When using the `computed({ get(k) { ... }, set(k , v) { ... } })` form, the setter method needs to return a value as well, as per the [API docs](https://emberjs.com/api/ember/2.18/classes/@ember%2Fobject/methods/computed?anchor=computed): 

> The `set` function should accept two parameters, `key` and `value`. The value returned from `set` will be the new value of the property.

By prepending `return` to `this.set(k, v)` you return the value passed to it, since [`this.set(k, v)` returns `v`](https://emberjs.com/api/ember/2.18/classes/@ember%2Fobject/methods/set?anchor=set).